### ISSUE TITLE
Synchronization method does not factor in starting subseason

### ISSUE THREAD

**Issue by Car-repo** [123oro321/sereneseasonsfix#8]
Running the latest Fix 1.1.0.1 for Serene Seasons 1.16.5-4.0.1.130.

I play starting in late spring with 4 day long subseasons. Currently on the 10th MC day, "/season time sync" sets calendar to late spring, whereas the correct season should be around mid summer. Clearly subseason length is considered, but no the initial subseason.

I have to add that I downloaded the Fix just yesterday night. Havent test it extensively, however, I still experience some 20-30 tick divergence upon reloading a world, which the mod should have fixed?

**Comment by 123oro321** [comment]
Hey. Currently sync sets season time to world time without taking into consideration the starting season from server configuration, I will add it to my to do list. For now you should change the season with the other commands, `/season time add 24d` will skip to next season (with default season length), or use `/season set SEASON`.
About the divergence, please confirm with `/season time info` the delta is changing after loading the world. Are there other mods? Is it happening always when you load the world?

**Comment by Car-repo** [comment]
Yes, delta changes whenever I load a world (regardless of whether I quit to desktop or just to main menu) even when only both Seasons mods are enabled. In fact, newly created worlds are already desynced for me. 

At least your mod presents a much needed way to properly synchronize clocks, great!

**Comment by 123oro321** [comment]
I am trying to reproduce the issue, I set the configuration to start at a different season and tried unloading/reloading the world few times. The delta stays the same no matter what I do.
Please let me know if you are using other mods or running the commands from different dimensions (by default only the overworld has seasons, nether/end's season time is fixed so the delta will naturally increase there)

**Comment by Car-repo** [comment]
Minecraft 1.16.5, Fix 1.1.0.1, Serene Seasons 4.0.1.130, Forge 36.2.42, Java 8 Update 421.

![2024-10-23_19 16 16](https://github.com/user-attachments/assets/7a4ccec3-b361-4c90-99c0-485080867d2b)
![2024-10-23_19 16 39](https://github.com/user-attachments/assets/7a33c4f1-ed68-4138-a790-1d34eabaa0d5)
![2024-10-23_19 17 22](https://github.com/user-attachments/assets/e64b81e4-2e00-457e-b56f-7afab34ee570)

I included Xaero's minimap to show real time hour. First one is upon world creation. I just run the command and exit. No other mod was enabled. Divergence is there but it's tiny and not a big deal for me anymore; other mods give me more pain.

**Comment by 123oro321** [comment]
1.1.1 supports subseasons. In order to correct the start time in existing worlds use /season time set_start
